vue/sbc-common-components/src/components/SbcSystemAlert.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <sbc-system-banner
+    v-if="isSbcSystemDown"
+    v-bind:show="isSbcSystemDown"
+    type="warning"
+    v-bind:message="alertMessage"
+  ></sbc-system-banner>
+</template>
+
+<script lang='ts'>
+import VueI18n from 'vue-i18n'
```

Comment:
I will see the package size after including vueI18n ... If its harmless , will go for including it.. otherwise we might have to remove it